### ISSUE TITLE
Text color in Markdown elements

### ISSUE THREAD

**Issue by GuilleHoardings** [flet-dev/flet#1303]
**Problem to Be solved**

If a `Markdown` element is used in a container with dark background, the text appears in black (if using the light Theme) and it is difficult to read.

**Suggested solution**

`Markdown` should have a `color` property for the text foreground, as `Text` elements do.

**Comment by FeodorFitsner** [comment]
Could you provide a repro code sample please?

**Comment by Zvetkov** [comment]
@GuilleHoardings is talking about blockquote syntax.
"> something"
This will not work normally in dark theme.
![image](https://user-images.githubusercontent.com/19334662/232575604-85291985-692b-47fe-b209-da58a8fc9242.png)
![image1](https://user-images.githubusercontent.com/19334662/232575606-ce7721a5-e604-4b2a-b871-544de23b92a1.png)

**Comment by Zvetkov** [comment]
I think good solution would have been to default to PRIMARY_CONTAINER or SECONDARY_CONTAINER for block quotes and to the relevant ON_... colors for text on blocks. 

Or to allow some customization for Markdown colors, including styling this and md font and color.

**Comment by GuilleHoardings** [comment]
In fact, my problem is with the light theme, the complementary problem to the one shown in the message above. I want to have some markdown inside a container that has a dark background when using the light theme, but if I do that, then I can't read the text. This is a minimum example:

```python
import flet as ft

def main(page: ft.Page):
    page.add(
        ft.Container(
            content=ft.Markdown(
                "I can't read this!",
                # color="#ffffff", # This is what I want to do
            ),
            bgcolor="#550000",
            padding=20,
        )
    )

ft.app(target=main, view=ft.WEB_BROWSER)
```

This is what I get:

![image](https://user-images.githubusercontent.com/19565386/232620434-0bca3190-da64-400d-aa49-0a022a57084b.png)

I would like to have the text in white (or any color that I select with `color`).

**Comment by FeodorFitsner** [comment]
It's been implemented in Flet 0.7.1.
